src/app/register/register.component.html

```diff
@@ -42,6 +42,70 @@
 
         </app-verify-email>
 
+      } @else if (this.emailVerificationToken?.valid) {
+        <p>
+          For a smoother giving experience create a Big Give account. Your information will be saved securely, making
+          future donations quicker and easier.
+        </p>
+
+        <p>
+          <strong>Your payment card</strong> will be saved for reuse, if you selected "
```

Comment:
Seems like this opening quote would show more normally and look better on the next line. Not sure if that's also an issue in the dialogs or if something about the Angular Material styling makes it work better there.